### PR TITLE
Stick input: Fix issue where deadzone noise from one device could drown out signal from another.

### LINKED ISSUE #15465: Analog Input via Keyboard is Being Dropped While Controller is Detected
### Game or games this happens in

None

### What area of the game / PPSSPP

Since v1.12.0, the analog input doesn't work correctly via keyboard while a controller is detected.
Not sure how to describe it, but it feels like it registers input on every odd frame but drops it on every even frame.
At least this is what it looks like in the calibration menu and on the touch control display.
In games, barely any of the input is recognized.

The issue disappears as soon as I disconnect my controller.

### What should happen

Analog input via keyboard should work smoothly even while a controller is detected, like it used to up until v1.11.3.

### Logs

In the video below you can see me input analog up, analog up+left and finally analog down+right.

https://user-images.githubusercontent.com/22328803/161383313-c5b4b57d-627a-4d17-83fc-ab58f60a0f81.mp4

### Platform

Windows

### Mobile phone model or graphics card

NVIDIA GeForce GTX 1060 3GB

### PPSSPP version affected

v1.12.x

### Last working version

v1.11.3

### Graphics backend (3D API)

Other

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try resetting settings or older versions and include if the issue is related.
- [X] Try without any cheats and without loading any save states.
- [X] Include logs or screenshots of issue.

### LINKED ISSUE #15992: Investigate defaults for Xbox controllers
I just paired an Xbox Series S/X controller to my Pixel with Android 12, and it seems to have the analog R trigger mapped to Gas+ instead of the expected TriggerR+, which I think is what I've seen before. We might have to tweak our defaults. Need to try pairing to various Android versions...

### PR DESCRIPTION
**Pull request by hrydgard**
Additionally, adds little trashcan icons to control mapper, replacing the "X" buttons.

Fixes #15465 

Fixes part of #16266 (part of it was already fixed)

Also fixes #15992

### CODE REVIEW

File: UI/ControlMappingScreen.cpp [hunk 1]
```diff
@@ -583,6 +583,7 @@ void JoystickHistoryView::Update() {
 
 AnalogSetupScreen::AnalogSetupScreen() {
 	mapper_.SetCallbacks([](int vkey) {}, [](int vkey) {}, [&](int stick, float x, float y) {
+		NOTICE_LOG(COMMON, "analog value from mapper: %f %f", x, y);
```
Comment (unknownbrackets) [inline UI/ControlMappingScreen.cpp:586]: Oops, debugging.

-[Unknown]

File: Core/ControlMapper.cpp [hunk 2]
```diff
@@ -49,24 +50,48 @@ void ControlMapper::SetRawCallback(std::function<void(int, float, float)> setRaw
...
+	bool ignore = false;
+
+	if (inDeadZone && lastNonDeadzoneDeviceID_[stick] != device) {
+		// Ignore this event! See issue #15465
+		NOTICE_LOG(COMMON, "Ignoring deadzone event from device %d (%d, %f)", device, axis, value);
+		ignore = true;
+	}
+
+	if (!inDeadZone) {
+		NOTICE_LOG(COMMON, "Got a non deadzone event from device %d (%d, %f > %f)", device, axis, value, g_Config.fAnalogDeadzone);
```
Comment (unknownbrackets) [inline Core/ControlMapper.cpp:82]: This also seems kinda spammy to be NOTICE?

-[Unknown]

Comment (hrydgard) [reply Core/ControlMapper.cpp:82]: Yeah, removing.

File: UI/ControlMappingScreen.cpp (hunk 1 above)
Comment (hrydgard) [reply UI/ControlMappingScreen.cpp:586]: Forgot, heh.

**Pull request merged**